hack/chains/rekor-verify-taskrun.sh

```diff
@@ -32,24 +32,34 @@ echo $TRANSPARENCY_URL
 pause
 
 title "Take a look at it"
-curl-json $TRANSPARENCY_URL | yq e . -P -
+curl-json $TRANSPARENCY_URL | yq-pretty
 pause
 
+# We could use rekor-cli here and it should be the same, e.g:
+# `rekor-cli get --log-index $LOG_INDEX --format json | jq ...`
+#
+BODY_DATA=$( curl-json $TRANSPARENCY_URL | jq -r 'values[].body' )
```

Comment:
Should we use `values[0]` here and in the line below since this script assumes a single value?

Reply:
It's probably safe since I think there's only one key/value, but I think you're right for the sake of readability and explicitness. I'll change it.

Reply:
Hmm, actually putting [0] there doesn't work... Will figure it out later..